tools-local/init-build.proj

```diff
@@ -105,12 +105,10 @@
   <Target Name="ExtractToolPackage"
           DependsOnTargets="BuildTasks"
           Condition="'$(OfflineBuild)' == 'true'">
-          <Message Importance="High" Text="Arcversion ->> $(ArcadeSdkOverride.Version)    SBPackPath ->> $(SourceBuiltPackagesPath)" />
     <ItemGroup>
       <_ToolPackage
-        Condition="'%(ArcadeSdkOverride.Version)' != ''"
-        Include="$(SourceBuiltPackagesPath)%(ArcadeSdkOverride.Identity).%(ArcadeSdkOverride.Version).nupkg"
-        Exclude="$(SourceBuiltPackagesPath)%(ArcadeSdkOverride.Identity).%(ArcadeSdkOverride.Version).symbols.nupkg"
+        Include="$(SourceBuiltPackagesPath)%.1.0.0-dev.nupkg"
+        Exclude="$(SourceBuiltPackagesPath)%.1.0.0-dev.symbols.nupkg"
         Id="%(ArcadeSdkOverride.Identity)" />
     </ItemGroup>
 
```

Comment:
For the ZipFileExtractToDirectory task, the destination should be "$(PackagesDir)microsoft.dotnet.arcade.sdk/1.0.0-dev/".

Also, I'm not sure you need the _ToolPackage property.  You could just specify the SourceArchive here.  "$(PrebuiltSourceBuiltPackagesPath)Microsoft.DotNet.Arcade.Sdk.1.0.0-dev.nupkg"

Reply:
I think the second part of this comment still applies.  I don't think you need the _ToolPackage property.